charmhelpers/contrib/network/ovs/__init__.py

```diff
@@ -410,6 +434,12 @@ def check_for_eni_source():
         eni.write('\nsource /etc/network/interfaces.d/*')
 
 
+def check_for_eni_source():
+    """DEPRECATED: call setup_eni() instead."""
+    log('DEPRECATION WARNING: call setup_eni() instead.')
```

Comment:
There's a handy decorator called `@deprecate(...)` from charmhelpers (i.e. from charmhelpers import deprecate) that provides deprecation warnings, etc.  It can also do single use, etc.